temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -652,3 +653,95 @@ message GetUsageResponse {
     // The next page's token.
     string next_page_token = 2;
 }
+
+message CreateExportSinkRequest {
```

Comment:
Can we add a `Namespace` qualifier to all the request, responses, apis to make it clear that these are namespace export sinks. 

Essentially this becomes `CreateNamespaceExportSinkRequest`